### ISSUE TITLE
Build failing due to lack of git clone permission

### ISSUE THREAD

**Issue by AnoRebel** [barrett-ruth/import-cost.nvim#6]
### Info

- Operating System: **Mint 21.1**
- `nvim --version`: **0.8.2 and nightly**
- Node package manager: **yarn**
- `yarn --version`: **1.22.19**

### Configuration

```lua
require('import-cost').setup ()
```

```lua
require('nvim-package-manager').setup {
    -- Any special plugin config
    {
        'barrett-ruth/import-cost.nvim',
        build = "sh install.sh yarn",
        config = function()
            require("import-cost").setup()
        end
    }
}
```

</details>

### Description

Basically the `wix/import-cost` clone in the `install.sh` uses the `git` protocol(I tried running the script manually) which requires one to have `ssh` access to git. Using the `https` protocol would make it it universal. Simply changing that should fix the bug. Unless there is something I am missing or there is a reason for it being specifically `git@github.com:wix/import-cost` rather than `https://github.com/wix/import-cost.git`

**Comment by barrett-ruth** [comment]
My SSH bias was showing. Personal question hy aren't you using it on your clones?

Still, fixed it anyway. The install script now defaults to HTTP

**Comment by AnoRebel** [comment]
Well, I used to, but I lost my PC, so when I got a temporary new one, and also having a work PC, I found it tedious to handle multiple ssh keys. But I guess its just plain procrastination(I keep telling myself I'll switch when I buy my new/desired PC). But I figured this fix would help even for those who want a quick setup in a VM or temporary PC where they don't need to setup SSH for it.

Thanks for the quick fix.
